src/IntlMoneyFormatter.php

```diff
@@ -0,0 +1,52 @@
+<?php
+namespace Money;
+
+final class IntlMoneyFormatter implements MoneyFormatter {
+
+    /**
+     * @var string
+     */
+    private $locale;
+    /**
+     * @var int
+     */
+    private $digits;
+
+    /**
+     * IntlMoneyFormatter constructor.
+     * @param int $digits
+     * @param string $locale
+     */
+    public function __construct($locale, $digits = 2)
```

Comment:
Is digit the same what I called subunit? If so I would rather call it fractionDigits following NumberFormatter convention. Also, I would not set it to 2 as default. Maybe it is common, but it depends on the user: one might not even support subunits (like rounds up cents and doesn't care about the rest)